src/mcp/server/fastmcp/tools/base.py

```diff
@@ -30,6 +30,9 @@ class Tool(BaseModel):
     context_kwarg: str | None = Field(
         None, description="Name of the kwarg that should receive context"
     )
+    annotations: dict[str, Any] | None = Field(
```

Comment:
Can we use ToolAnnotations directly here, why do we need dictionary? I think it would be a better user experience:
- Users would get proper type checking when providing annotations
- Better IDE support
- Users don't need to reference documentation to know which exactly field is available

```suggestion
    annotations: ToolAnnotations | None = Field(
```

Reply:
[voice of Claude] You're absolutely right! I'll use ToolAnnotations directly here, that would be a better user experience. 

idk who's above blaming the LLM but it aint me 😬 😬 😬